data/json/proficiencies/melee_weapons.json

```diff
@@ -0,0 +1,778 @@
+[
+  {
+    "type": "proficiency",
+    "id": "prof_auto_rifles_familiar",
+    "//": "Maybe scrap this.",
+    "category": "prof_combat",
+    "name": { "str": "Automatic Rifle Familiarity" },
+    "description": "You're familiar with how to hold an automatic rifle, use a bayonette, and not cut yourself.  In case you need to club someone with it instead of just shooting them, I guess.",
```

Comment:
```suggestion
    "description": "You're familiar with how to hold an automatic rifle, use a bayonet, and not cut yourself.  In case you need to club someone with it instead of just shooting them, I guess.",
```